src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -48,6 +60,46 @@ public void init() throws ServletException {
   void setUserStore(UserStore userStore) {
     this.userStore = userStore;
   }
+  
+  /**
+   * Sets the ConversationStore used by this servlet. This function provides a common setup method
+   * for use by the test framework or the servlet's init() function.
+   */
+  void setConversationStore(ConversationStore conversationStore) {
+    this.conversationStore = conversationStore;
+  }
+  
+  /**
+   * Sets the MessageStore used by this servlet. This function provides a common setup method
+   * for use by the test framework or the servlet's init() function.
+   */
+  void setMessageStore(MessageStore messageStore) {
+    this.messageStore = messageStore;
+  }
+  
+  /**Gets the user who has sent the most messages. If there is a tie, the user who registered
+   * first is given.
+   * */
+  String mostActiveUser() {
+	  
```

Comment:
This indent isn't consistent with the indents above. Might as well indent all of it to Google style (4 spaces, no tabs ever)